### PR TITLE
Transit instructions re #6

### PR DESCRIPTION
**Pull request by WorldsEndless**
As per #6 , I have added a readme section that includes an example of using deja-fu with Transit.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -263,6 +263,31 @@ issue. Note that you'll have to set these up yourself, since using
...
+
+```clojure
+(ns deja-foo
+  (:require [cognitect.transit :as transit]
+            [lambdaisland.deja-fu :as fu]
+            [goog.date.Date]))
+
+(let [auth-write-handler {:handlers {goog.date.Date
+                                       (transit/write-handler
+                                        (constantly "LocalDate")
```
Comment (plexus) [inline README.md:277]: `"LocalDate"`, or `"time/date"`? the comment at the end suggests the latter.

File: README.md [hunk 2]
```diff
@@ -263,6 +263,31 @@ issue. Note that you'll have to set these up yourself, since using
...
+                                        fu/format)}}
+        auth-read-handlers  {:handlers {"LocalDate"
+                                        (transit/read-handler 
+                                         fu/parse-local-date)}}
+        treader (transit/reader. "json" auth-read-handlers)
+        twriter (transit/writer. "json" auth-write-handler) ]
```
Comment (plexus) [inline README.md:283]: I don't think the periods are necessary here? `"json"` can be a keyword instead: `:json`

File: README.md [hunk 3]
```diff
@@ -263,6 +263,31 @@ issue. Note that you'll have to set these up yourself, since using
...
+(ns deja-foo
+  (:require [cognitect.transit :as transit]
+            [lambdaisland.deja-fu :as fu]
+            [goog.date.Date]))
+
+(let [auth-write-handler {:handlers {goog.date.Date
+                                       (transit/write-handler
+                                        (constantly "LocalDate")
+                                        fu/format)}}
+        auth-read-handlers  {:handlers {"LocalDate"
```
Comment (plexus) [inline README.md:279]: Seems the indentation is off here.

Comment (WorldsEndless) [reply README.md:279]: Hmm... that's straight from my editor Clojure mode, uncustomized

**Comment by alysbrooks** [comment]
@WorldsEndless No rush, but do will you be able to take a look at this soon?

**Comment by WorldsEndless** [comment]
Hmm... I never understood this was waiting for me. Changes requested are unclear to me.

**Comment by plexus** [comment]
Hey, my apologies for not being more clear. I guess it's not really obvious from the inline diff, but this is what it looks like on your branch.

```clojure
(let [auth-write-handler {:handlers {goog.date.Date
                                       (transit/write-handler
                                        (constantly "LocalDate")
                                        fu/format)}}
        auth-read-handlers  {:handlers {"LocalDate"
                                        (transit/read-handler 
                                         fu/parse-local-date)}}
        treader (transit/reader :json auth-read-handlers)
        twriter (transit/writer :json auth-write-handler) ]
    (->> (fu/local-date)
         (transit/write twriter)
         (transit/read treader))
    );; => #time/date2021-10-11
```

I would expect `auth-write-handler`, `auth-read-handlers` and so forth to line up, and to have two spaces of indent inside the let instead of four.

I'll merge now and fix it up, then we're good to go! Thanks for the contribution!

**Pull request merged**